### PR TITLE
Update eth_getBalance to use mirror node

### LINKED ISSUE #448: Update `eth_getBalance` with logic to call mirror node account endpoint
### Problem

Mirror node added logic tot eh `api/v1/accounts/{accountIdOrEvmAddress}` to return the up to date balance.
However, the relay still calls the consensus node

### Solution

Update `eth_getBalance` with logic to call mirror node account endpoint

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by Nana-EC**
Signed-off-by: Nana Essilfie-Conduah <nana@swirldslabs.com>

**Description**:
`eth_getBalance` currently uses the consensus nodes to retrieve balance since mirror node only update balance every 15 mins.
With mirror node fix to update on transfers relay can now point to mirror node

- Update `eth_getBalance` logic to call Mirror Node /api/v1/accounts` endpoint and extract balance
- Fix metric discrepancy on `resolvedCost`
- Update sdkClient flow to remove unnecessary error check since default is UNKNOWN

**Related issue(s)**:

Fixes #448 

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: packages/relay/src/lib/eth.ts [hunk 1]
```diff
@@ -507,25 +507,18 @@ export class EthImpl implements Eth {
```
Comment (georgi-l95) [inline packages/relay/src/lib/eth.ts:507]: 10 lines above this, there is a comment, which I think can be removed now with this PR.

**Review by georgi-l95 — CHANGES_REQUESTED** [review]
LG, but one acceptance test is failing.
`@release should execute "eth_getBalance" for newly created account with 10 HBAR`

File: packages/relay/src/lib/eth.ts (hunk 1 above)
Comment (Nana-EC) [reply packages/relay/src/lib/eth.ts:507]: Thanks

**Review by georgi-l95 — APPROVED** [review]
LGTM

**Review by lukelee-sl — APPROVED** [review]

**Pull request merged**